modules/javamail/src/main/scala/emil/javamail/conv/BodyDecode.scala

```diff
@@ -37,7 +37,14 @@ trait BodyDecode {
         val filename = Option(bp.getFileName)
         val bytes = BodyDecode.loadPart(bp)
         val (len, dataAsStream) = (bytes.length, Stream.chunk(Chunk.byteVector(bytes)))
-        Attachments(Attachment(filename, mt, dataAsStream, len.pure[F]))
+        val contentId = Option(bp.getHeader("Content-ID"))
+          .flatMap(a => a.headOption)
+          // All IDs (content, mail, ...) are wrapped in '<' '>'
+          .map(s => s.drop(1).dropRight(1))
```

Comment:
I myself was wrong so many times with these assumptions with e-mails :) - perhaps its good to remove only if it is really a `<` or `>` - wdyt? (also a nitpicking comment)

Reply:
Oh yes, the RFC is clear about it, but maybe we'll see mail in the wild without < >. Together with the multipart/related stuff, I'll do a small rework here